config/environments/development.rb

```diff
@@ -5,7 +5,8 @@
   # every request.  This slows down response time but is perfect for development
   # since you don't have to restart the webserver when you make code changes.
   config.cache_classes = false
-  config.cache_store = :memory_store
+  config.cache_store = :dalli_store, '127.0.0.1:11211',
+                          { :namespace =>  'ao3-v1', :expires_in =>  0, :compress => true , :pool_size => 10 }
```

Comment:
Just checking, is this going to work on vagrant or does the vagrant need to be updated?